Project.toml

```diff
@@ -1,13 +1,31 @@
 name = "ADTypes"
 uuid = "47edcb42-4c32-4615-8424-f2b9edc5f35b"
-authors = ["Vaibhav Dixit <vaibhavyashdixit@gmail.com> and contributors"]
-version = "0.2.7"
+authors = [
+    "Vaibhav Dixit <vaibhavyashdixit@gmail.com>, Guillaume Dalle and contributors",
+]
+version = "0.3.0"
```

Comment:
```suggestion
version = "1.0.0"
```

Reply:
Not sure if this is needed right away, can you explain why you'd prefer to skip to 1 directly?

Reply:
Pre 1.0 versioning has issues so if we're doing a breaking change then we should go to 1.0

Reply:
I think this design is sound so I would agree. And nothing stops us from going beyond v1 in the future, even if many packages are reluctant to do so